frontend/src/lib/components/BillingAlertsV2.tsx

```diff
@@ -1,7 +1,7 @@
 import { useActions, useValues } from 'kea'
 import { router } from 'kea-router'
 import { useEffect, useState } from 'react'
-import { billingLogic } from 'scenes/billing/v2/billingLogic'
+import { billingLogic } from 'scenes/billing/v2/control/billingLogic'
```

Comment:
I think the logic shouldn't be in the control folder (in fact I think you duplicated it there by mistake, as it's also in the folder above)

Reply:
There were two `billingLogic`s in the first place. They appear to be different, one is for the V2 and one is for V1. I wasn't sure if i was going to need to change things in the logic so I duplicated the `v2/billingLogic` into `v2/test/billingTestLogic`.